doc/specs/SCOOBE.md

```diff
@@ -0,0 +1,185 @@
+
+**2.2. Expected Impact: Customer, and Technology Outcomes**
+
+- **High Reliability** : Less than 0.1% crash rate.
+- **Increased Activation** : 50% or more adoption rate of new feature/utilities among PowerToys users who already utilize associated tools.
+- **High User Retention:** 25% or more active PowerToys users after 28 days of upgrade.
+
+## 3. Requirements
+
+The SCOOBE dialog builds off the currently implemented OOBE dialog originally drafted by [Niel&#39;s mock-up.](https://github.com/microsoft/PowerToys/issues/1285)
+
+**3.1. Functional Requirements**
+
+**3.1.1. Functional Requirements**  **Overview**
+
+| **No.** | **Requirement** | **Pri** |
+| --- | --- | --- |
+|1. | The SCOOBE dialog should launch immediately after PowerToys is updated. | P0 |
+|2. | The SCOOBE dialog should be contained inside the existing OOBE Dialog under its own &quot;What&#39;s New&quot; page of the dialog window. See figure 5.1.1. | P0 |
+|3. | The content for the SCOOBE dialog should be stored externally from the PowerToys application. **\*** | P0 |
+|4. | When the &quot;What&#39;s New&quot; page is opened, the content displayed should by loaded dynamically from the relevant information contained in the external storage described in 3.1.1.3 above. Assumes the user&#39;s device is connected to the internet. | P0 |
+|5. | The SCOOBE dialog should display information on updates that have occurred between the previous version of PowerToys the user had installed, and the current version of PowerToys the user has updated to. **\*\*** | P0 |
+|6. | If PowerToys was installed for the first time, the SCOOBE dialog should only display information on updates that have occurred with the version of PowerToys the user has installed. | P0 |
+|7. | If PowerToys was installed for the first time, the OOBE&#39;s &quot;Welcome to PowerToys&quot; page should display first, not the SCOOBE&#39;s &quot;What&#39;s New&quot; page. See figure 5.1.2. | P0 |
+|8. | The structure of the SCOOBE dialog page&#39;s content should follow the guidelines described in section 3.1.2. | P0 |
+|9. | After SCOOBE is initially viewed, the user should be able to re-access the SCOOBE dialog at any time by opening the OOBE window again and selecting the &quot;What&#39;s New&quot; page. | P1 |
+
+**\*** - By storing the content for SCOOBE externally from the application, the PowerToys team can update/adjust information without being constrained to PowerToys&#39; release cycles. This is critical in the event of errors or miscommunications that would otherwise be difficult, if not impossible, to correct if stored locally to the app and shipped with the version of PowerToys being released.
```

Comment:
Any plans to caching the data for the case that no network is available when the user reopens the oobe section?

Reply:
good call out

Reply:
This point also applies to OOBE, but as explained in previous comment, it's important to keep them in sync. The ability to update just SCOOBE is bring little to no benefits to the users, but brings more complexity for development and testing and opens the door to inconsistencies with OOBE.